File: M80AttributedLabel/Classes/Demo/BasicAttributedLabelViewController.m
//
//  BasicAttributedLabelViewController.m
//  M80AttributedLabel
//
//  Created by amao on 5/21/14.
//  Copyright (c) 2014 www.xiangwangfeng.com. All rights reserved.
//

#import "BasicAttributedLabelViewController.h"
#import "M80AttributedLabel.h"

@interface BasicAttributedLabelViewController ()

@end

@implementation BasicAttributedLabelViewController

- (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil
{
    self = [super initWithNibName:nibNameOrNil bundle:nibBundleOrNil];
    if (self) {
        
    }
    return self;
}

- (void)viewDidLoad
{
    [super viewDidLoad];
    
    M80AttributedLabel *label = [[M80AttributedLabel alloc]initWithFrame:CGRectZero];
    
    label.text      = @"Hello M80AttributedLabel";
    label.font      = [UIFont fontWithName:@"Zapfino" size:25];
    label.textColor = UIColorFromRGB(0xFF9F00);
    label.shadowColor = [UIColor grayColor];
    label.shadowOffset= CGSizeMake(1, 1);
    label.shadowBlur = 1;
    
    label.frame     = CGRectInset(self.view.bounds,20,20);

    [self.view addSubview:label];
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
}



@end

File: SwiftDemo/Classes/BasicAttributedLabelViewController.swift
//
//  BasicAttributedLabelViewController.swift
//  SwiftDemo
//
//  Created by amao on 2016/10/12.
//  Copyright © 2016年 amao. All rights reserved.
//

import UIKit

class BasicAttributedLabelViewController: BaseViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        self.title = "Basic"

        let label = M80AttributedLabel()
        label.text = "Hello M80AttribtuedLabel"
        label.font = UIFont(name: "Zapfino", size: 25)
        label.textColor = RGB(0xFF9F00)
        label.shadowColor = UIColor.gray
        label.shadowBlur = 1
        label.frame = self.view.bounds.insetBy(dx: 20, dy: 20)
        
        self.view.addSubview(label)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    


}
